pkg/cmd/pulumi/import.go

```diff
@@ -224,6 +240,21 @@ func parseImportFile(f importFile, protectResources bool) ([]deploy.Import, impo
 			pusherrf("%v has no ID", describeResource(i, spec))
 		}
 
+		// Make spec.Name unique.
+		name := spec.Name
+		_, isDuplicate := duplicateIndexes[i]
```

Comment:
this can just look in duplicateNames

Reply:
It could, but the first instance that we don't want to suffix by making it unique would think it's a duplicate as well.